### PR TITLE
Slider widget

### PR DESCRIPTION
**Pull request by alexander-daniel**
screen shot 2018-05-08 at 9 46 03 am

Super simple to get us going, basics of sass overriding so it looks like how we want it to!

Usage will have to define width, hence the 100% here

### CODE REVIEW

**Comment by mikevespi** [comment]
cool 🚚

File: src/scss/views/widgets/_slider.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Here we import the base styles from rc-slider module, and we overwrite/extend
+ * what we need to, to make it look and feel how we want.
+ */
+@import "../../node_modules/rc-slider/assets/index.css";
```
Comment (Sundiata2) [inline src/scss/views/widgets/_slider.scss]: neat

**Comment by Sundiata2** [comment]
🚚

File: src/scss/views/widgets/_slider.scss (hunk 1 above)
Comment (alexander-daniel) [reply src/scss/views/widgets/_slider.scss]: until they change the path, but yeah, pretty neat lol

**Pull request merged**